tomcat-base/src/shared/main/java/eu/xenit/alfresco/tomcat/embedded/utils/Utils.java

```diff
@@ -31,17 +31,38 @@ public static void redirectLog4j(Path webappPath, Path destinationPath) {
         try {
             try (var reader = Files.newBufferedReader(log4JPropertiesPath)) {
                 properties.load(reader);
+                // log4j 1.x properties
                 properties.setProperty("log4j.rootLogger", "error, Console, jmxlogger1");
                 properties.setProperty("log4j.appender.Console.layout", "eu.xenit.logging.json.log4j.JsonLayout");
                 properties.setProperty("log4j.appender.Console.layout.Type", "application");
                 properties.setProperty("log4j.appender.Console.layout.Component", webappPath.getFileName().toString());
                 properties.setProperty("log4j.appender.Console.layout.ExtractStackTrace", "true");
                 properties.setProperty("log4j.appender.Console.layout.FilterStackTrace", "true");
+                // log4j 2.x properties
+                properties.setProperty("rootLogger.level", "error");
+                properties.setProperty("rootLogger.appenderRefs", "stdout");
+                properties.setProperty("rootLogger.appenderRef.stdout.ref", "ConsoleAppender");
+                properties.setProperty("appender.console.type", "Console");
+                properties.setProperty("appender.console.name", "ConsoleAppender");
+                properties.setProperty("appender.console.layout.type", "JsonTemplateLayout");
+                properties.setProperty("appender.console.layout.eventTemplateUri"
+                        , "file:///usr/local/tomcat/log4j2/jsonLayout/jsonLayout.json");
+                properties.setProperty("appender.console.layout.eventTemplateAdditionalField[0].type", "EventTemplateAdditionalField");
+                properties.setProperty("appender.console.layout.eventTemplateAdditionalField[0].key", "facility");
+                properties.setProperty("appender.console.layout.eventTemplateAdditionalField[0].value", "xenit-json");
```

Comment:
Not part of the XEP
https://xenitsupport.jira.com/wiki/spaces/XEN/pages/3657498642/XEP-21+Logging+formats

I would leave it out. It's only there because it was in the code we forked from.

Reply:
Ok, I will remove it.